py/plugins/dev-local-vector-store/src/genkit/plugins/dev_local_vector_store/indexer.py

```diff
@@ -0,0 +1,63 @@
+# Copyright 2025 Google LLC
+# SPDX-License-Identifier: Apache-2.0
+
+import asyncio
+import json
+from hashlib import sha256
```

Comment:
The JS version uses md5, can we use that? If possible we should make it compatible to reuse the same DB files (but it is not a blocker IMO).

Reply:
No, cryto security is not essential here. This is just a unique hash based on the document's content. Also, this plugin is not intended to be used in prod so rare collisions are acceptable.